chord_metadata_service/mohpackets/docs/schema.yml

```
@@ -2554,14 +1879,19 @@ components:
           - type: 'null'
         submitter_treatment_id:
           maxLength: 64
-          pattern: ^[A-Za-z0-9\-\._]{1,64}
+          pattern: ^[A-Za-z0-9\-\._]{1,64}$
           title: Submitter Treatment Id
           type: string
         surgeries:
           items:
             $ref: '#/components/schemas/NestedSurgerySchema'
           title: Surgeries
           type: array
+        systemictherapies:
```

```suggestion
        systemic_therapies:
```
I think it needs an underscore to be consistent with how we have done it for other schemas